apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1/ComputeEnumHelpers.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Protobuf.Reflection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Google.Cloud.Compute.V1
+{
+    /// <summary>
+    /// Helper methods and constants for conversion between the enums within the Compute
+    /// API and the string representations used on the wire.
+    /// </summary>
+    public static class ComputeEnumHelpers
+    {
+        /// <summary>
+        /// Attempts to parse the given string value as the wire representation of
```

Comment:
```suggestion
        /// Attempts to parse the given string value from the wire representation of
```

Reply:
Nope, this one was deliberate - it's saying "think of this string value as the wire representation". (We can fiddle later, of course.)